### DISCUSSION TITLE
Can't Get Verified - Can't Get An Answer

### DISCUSSION THREAD

**Discussion by jmbjacobs** [community/community#64617] [GitHub Education]
### Hi everyone,

Hello!  I have been trying to get verified as a teacher for a month now and can't get any response to two separate help tickets.  The GitHub Education account rules say I need a tarriers.org account, but those are reserved for students, not teachers.  I have explained this, but have not received a response (I have followed up multiple times).

Is there a phone number I can call?  I must have an educator GitHub account to use some features I learned about at the APAC in July.

TIA for any helpful advice/direction.

**Comment by LinukaAr** [comment]
I understand your frustration with the verification process for a teacher's GitHub Education. I can offer some general advice on how to proceed:

1. **Check GitHub Education Documentation:** Make sure you have thoroughly reviewed the GitHub Education documentation and requirements for teacher verification. This will help you ensure that you have provided all the necessary information and documentation.

2. **Check Spam Folder:** Sometimes, email responses can end up in spam folders. Double-check your email account's spam folder to make sure you haven't missed any responses from GitHub support.

3. **Follow Up Emails:** If you have already sent multiple help tickets and haven't received a response, consider sending follow-up emails to your existing tickets. Politely explain the situation again and express your eagerness to get the issue resolved.

4. **Social Media:** Some companies have responsive social media teams. You could try reaching out to GitHub's official social media accounts (e.g., Twitter) and explain the situation. They might be able to direct you to the appropriate channels for support.

5. **GitHub Community Forum:** The GitHub Community Forum is a place where developers, users, and GitHub staff engage in discussions. You might be able to get some advice from other users who have faced similar issues or find official GitHub staff responding to queries.

6. **GitHub Support Page:** Visit GitHub's official support page to see if they have any updated contact information or alternative methods for getting in touch with their support team.

7. **Patience and Persistence:** Sometimes, support response times can vary based on the volume of requests they receive. While waiting is frustrating, persistence in following up with them might eventually lead to a resolution.

8. **Contact via School or Institution:** If you're associated with an educational institution, you could also inquire if they have any channels for contacting GitHub support for educational accounts. Some institutions might have established contacts.

Remember that my information might not be up-to-date, so it's a good idea to visit GitHub's official website or support page for the most accurate and current contact information. I hope you're able to get your teacher verification issue resolved soon so that you can access the features you need for your educational activities.
